docs/cep/sink/sink-types/table/index.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 40
+title: Table (Collection)
+---
+
+A table is a stored version of an stream or a table of events. Its schema is defined in the _table definition_ that is similar to a stream definition. These events are stored in database. In Macrometa GDN, tables are called [collections](../../../../collections/index.md).
+
+## Purpose
+
+Tables allow the stream worker to work with stored events. By defining a schema for tables, the stream processor enables them to be processed by queries using their defined attributes with the streaming data. You can also query the state of the stored events in the table.
+
+Any table defined in a stream worker is automatically a sink for that stream worker.
+
+### Syntax
+
+The syntax for a new table definition is as follows:
+
+```sql
+CREATE TABLE (GLOBAL|SPOT)? <table_name> (<attribute_name> <attribute_type>, ...);
+```
+
+### Parameters
+
+The following parameters are configured in a table definition:
+
+| Parameter     | Description |
+| ------------- |-------------|
+| table name      | The name of the table defined. (`PascalCase` is used for table name as a convention.) |
+| attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
+| attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
+| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      | _from         | STRING              | Yes      |
+| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. | _to      | STRING              | Yes      |
+
+### Examples
+
+The following defines a local table named `RoomTypeTable` with `roomNo` and `type` attributes of data types `int` and `string` respectively.
+
+```sql
+CREATE TABLE RoomTypeTable (roomNo int, type string);
+```
+
+The following defines a global table named `SweetProductionCollection` with `name` and `amount` attributes of data types `string` and `double`.
```

Comment:
@diana-macrometa Just a general thought for our examples: This `SweetProduction` name pops up a lot, but it's just a generic example for passing data into a collection.  I think we should consider making it more generic to something like "ExampleProduction" and change the "sweets" to something more generic like "data."  Otherwise, we should make it more specific to an actual business use case.  

As it stands, "SweetProduction" is a bit vague and confusing for my liking.